### ISSUE TITLE
[A11Y] Account link dropdown toggle MUST have contrast ratio of at least 4.5:1

### ISSUE THREAD

**Issue by jenstrickland** [rubyforgood/voices-of-consent#434]
# Severity: Serious - WCAG 2.0 (AA): MUST

**Feedback Framework**
```diff
- MUST - must be addressed to be WCAG 2.0 AA compliant
! SHOULD - is an accessibility best practice
+ CONSIDER - things to think about to improve inclusive design and accessibility
```
## Description

The Account dropdown link color is #007bff on a background of #eeeeee, which has an insufficient color contrast ratio of 3.42:1. In order to meet WCAG 2.0 AA, color contrast **MUST** be a minimum of 4.5:1.

![Screen Shot 2020-03-18 at 7 52 41 PM](https://user-images.githubusercontent.com/57469/77017912-1fcd3b80-6952-11ea-82b0-64300b8357fb.png)

## Disabilities affected
 * Low Vision
 * Colorblindness

## Potential fix

Update link color from #007bff to #003be0, for a contrast ratio of 6.69:1.

## Success criterion

Ensure color contrast of at least 4.5:1 for small text or 3:1 for large text, even if text is part of an image. Large text has been defined in the requirements as 18pt (24 CSS pixels) or 14pt bold (19 CSS pixels). Note: Elements found to have a 1:1 ratio are considered "incomplete" and require a manual review.

## Why it matters

Some people with low vision experience low contrast, meaning that there aren't very many bright or dark areas. Everything tends to appear about the same brightness, which makes it hard to distinguish outlines, borders, edges, and details. Text that is too close in luminance (brightness) to the background can be hard to read.

There are nearly three times more individuals with low vision than those with total blindness. One in twelve people cannot see the average full spectrum of colors - about 8% of men and 0.4% of women in the US. A person with low vision or color blindness is unable to distinguish text against a background without sufficient contrast.

Color transparency and opacity is taken into account in the background.

Color transparency and opacity in the foreground is more difficult to detect and account for due to:

- 1:1 colors in foreground and background.
- CSS background gradients.
- Background colors in CSS pseudo-elements.
- Background colors created with CSS borders.
- Overlap by another element in the foreground - this sometimes comes up with tricky positioning.
- Elements moved outside the viewport via CSS.

## WCAG Guidance

* [WCAG 1.4.3 Text elements must have sufficient color contrast against the background](https://dequeuniversity.com/rules/axe/3.5/color-contrast?application=AxeChrome)
